### ISSUE TITLE
Won't work

### ISSUE THREAD

**Issue by ceenee** [CellularPrivacy/Android-IMSI-Catcher-Detector#982]
Is not compatible with Android 10

### Agreements

- [x] **I have reviewed and accepted the [guidelines for contributing](https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector/blob/development/.github/CONTRIBUTING.md) to this project.**
- [x] **I have searched the [open Issues](https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector/issues) and made sure I am not filing a duplicate.**
- [x] **I have read the notes on [debugging](https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector/blob/development/.github/CONTRIBUTING.md#debugging) and will properly maintain this Issue.**

### Overview

* `AIMSICD` version (see the `About` screen):
* Phone and ROM running *current* release:
* Link to logfile without identifyable data:

**Comment by veronikaverochka** [comment]
Doesn't work on Xiaomi Phones that have Android Version 10.
I am running 
xiaomi redmi note 8 pro
MIUI 12.0.3.0 Stable 
Android Version: 
10QP1A. 190711.020

Octo- core : 2.05 GHz

Ram: 6gb

Last updated on: Feb 13, 2021 at 6:40am est

**Comment by ghost** [comment]
Project is dead and dont work fully on any device

**Comment by flanneryb** [comment]
There was a update 4 months ago.  There is a War going on in Ukraine, so I think the primary developers are a little busy.

**Comment by ghost** [comment]
> There was a update 4 months ago

@flanneryb No) Last code update was in 2017

**Comment by flanneryb** [comment]
Yes, your right, I was looking at the src folder activity.  It has not been touched in over a year.

**Comment by drmatoi** [comment]
> There was a update 4 months ago.  There is a War going on in Ukraine, so I think the primary developers are a little busy.

Only translations

**Comment by flanneryb** [comment]
I didn't know that he was in the Ukraine.  Does he need anything?

**Comment by MrRobinnn** [comment]
Disable your MIUI from developer options on your phone
